File: prak5/PRAK105-2510817210003-MuhammadRaffaAzisha.py
a = 9
b = 5
x = 8
y = 8
sisa_ab = a % b
sisa_xy = x % y
total_sisa = sisa_ab + sisa_xy

print(f"Variabel a bernilai {a}")
print(f"Variabel b bernilai {b}")
print(f"Variabel x bernilai {x}")
print(f"Variabel y bernilai {y}\n")

print(f"Sisa bagi dari a / b ({a} % {b}) adalah {sisa_ab}")
print(f"Sisa bagi dari x / y ({x} % {y}) adalah {sisa_xy}\n")

print(f"Total sisa bagi dari a dibagi b dan x dibagi y adalah {total_sisa}")

File: prak5/PRAK105-2510817210003-MuhammadRaffaAzisha.c
#include <stdio.h>

int main() {
    int a = 9;
    int b = 5;
    int x = 8;
    int y = 8;

    int sisa_ab = a % b; 
    int sisa_xy = x % y; 
    int total_sisa = sisa_ab + sisa_xy;
    
    printf("Variabel a bernilai %d\n", a);
    printf("Variabel b bernilai %d\n", b);
    printf("Variabel x bernilai %d\n", x);
    printf("Variabel y bernilai %d\n", y);
    printf("\n"); // Baris baru untuk spasi

    printf("Sisa bagi dari a / b (%d %% %d) adalah %d\n", a, b, sisa_ab);
    printf("Sisa bagi dari x / y (%d %% %d) adalah %d\n", x, y, sisa_xy);
    printf("\n"); // Baris baru untuk spasi
    
    printf("Total sisa bagi dari a dibagi b dan x dibagi y adalah %d\n", total_sisa);

    return 0;
}